### PR TITLE
Add Debug Menu

### PR DESCRIPTION
**Pull request by iUsmanN-Guardian**
## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: Sources/Nophan/Debug/NophanMonitor.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//  NophanMonitor.swift
+//  
+//
+//  Created by Usman Nazir on 28/05/2024.
+//
+
+import Foundation
+
+/// A struct that maintains an array of the last 15 NophanRequest objects.
+internal class NophanMonitor: ObservableObject {
```
Comment (robindorpe-guardian) [inline Sources/Nophan/Debug/NophanMonitor.swift:11]: Why are you using `ObservableObject` instead of `@Observable`

**Review by robindorpe-guardian — APPROVED** [review]

File: Sources/Nophan/Debug/NophanMonitor.swift (hunk 1 above)
Comment (iUsmanN-Guardian) [reply Sources/Nophan/Debug/NophanMonitor.swift:11]: `@Observable` is available iOS 17 onwards?

Comment (robindorpe-guardian) [reply Sources/Nophan/Debug/NophanMonitor.swift:11]: yes

Comment (robindorpe-guardian) [reply Sources/Nophan/Debug/NophanMonitor.swift:11]: We use `@Observable` on Feast. Will Nophan be used on the main app as well? If yes then nevermind.

Comment (iUsmanN-Guardian) [reply Sources/Nophan/Debug/NophanMonitor.swift:11]: Yes, that's the plan.

**Pull request merged**